packages/story-editor/src/components/panels/design/videoAccessibility/karma/videoPoster.karma.js

```diff
@@ -105,11 +110,108 @@ describe('Video Accessibility Panel', () => {
+      await fixture.snapshot('Menu open');
+
+      // And click on edit
+      await fixture.events.click(vaPanel.posterMenuHotlink);
+
+      const input = fixture.screen.getByRole('textbox', { name: 'URL' });
+      const insertBtn = fixture.screen.getByRole('button', {
+        name: 'Use image as video poster',
+      });
+      await fixture.events.click(input);
+      await fixture.events.keyboard.type(
+        'http://localhost:9876/__static__/ranger9.jpg'
+      );
+
+      await fixture.events.click(insertBtn);
+      await fixture.events.sleep(500);
+      // Expect poster image to have updated ( See MediaUpload component in fixture.js )
+      expect(vaPanel.posterImage.src).toMatch(/^http.+\/ranger9.jpg$/);
+
+      // Now open menu and click reset
+      await fixture.events.click(vaPanel.posterMenuButton);
+      await fixture.events.click(vaPanel.posterMenuReset);
+
+      // Expect poster image to have been reset
+      expect(vaPanel.posterImage.src).toBe(originalPoster);
+    });
+
+    it('should not be allowed to hotlink images with incorrect dimensions', async () => {
+      vaPanel.posterMenuButton.scrollIntoView();
+
+      // Expect menu button to exist
+      expect(vaPanel.posterMenuButton).toBeTruthy();
+
+      // Open the menu
+      await fixture.events.click(vaPanel.posterMenuButton);
+      await fixture.snapshot('Menu open');
+
+      // And click on edit
+      await fixture.events.click(vaPanel.posterMenuHotlink);
+
+      const input = fixture.screen.getByRole('textbox', { name: 'URL' });
+      const insertBtn = fixture.screen.getByRole('button', {
+        name: 'Use image as video poster',
+      });
+      await fixture.events.click(input);
+      const hotlinkUrl = 'http://localhost:9876/__static__/saturn.jpg';
+      await fixture.events.keyboard.type(hotlinkUrl);
+
+      await fixture.events.click(insertBtn);
+      await fixture.events.sleep(500);
+      const dialog = screen.getByRole('dialog');
+      await waitFor(() =>
+        expect(dialog.textContent).toContain(
+          'do not match required image dimensions'
+        )
+      );
+    });
+
+    it('should not be allowed to hotlink images with invalid url', async () => {
+      vaPanel.posterMenuButton.scrollIntoView();
+
+      // Expect menu button to exist
+      expect(vaPanel.posterMenuButton).toBeTruthy();
+
+      // Open the menu
+      await fixture.events.click(vaPanel.posterMenuButton);
+      await fixture.snapshot('Menu open');
+
+      // And click on edit
+      await fixture.events.click(vaPanel.posterMenuHotlink);
+
+      const input = fixture.screen.getByRole('textbox', { name: 'URL' });
+      const insertBtn = fixture.screen.getByRole('button', {
+        name: 'Use image as video poster',
+      });
+      await fixture.events.click(input);
+      const hotlinkUrl = 'invalid';
+      await fixture.events.keyboard.type(hotlinkUrl);
+
+      await fixture.events.click(insertBtn);
+      await fixture.events.sleep(500);
```

Comment:
Here and elsewhere -- is the `sleep` for waiting until the dialogue appears?
Would be better to use `waitFor` instead for waiting as much as needed only, for example something like this:
```
      await waitFor(() => {
        dialog = screen.getByRole('dialog');
        if (!dialog) {
          throw new Error('dialog not ready');
        }
        expect(dialog.textContent).toContain('Invalid link')
      });
```